.github/workflows/npm-publish.yml

```diff
@@ -14,9 +14,18 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: 12
+          node-version: 14
       - run: npm ci
-      - run: npm test
+      - name: Mock testing
```

Comment:
I'm uncertain here. Is it mock testing (integration), or unit testing ❓

Reply:
Well by sure it's involving a mock (in opposite to end to end testing), but it's just a label, it can be changed as wished.